src/components/header/Header.jsx

```diff
@@ -10,12 +10,15 @@ export class Header extends React.Component {
     this.state = {
       userDropdown: false,
       locationDropdown: false,
+      logoLinkUrl: "../../referenceapplication/home.page",
+      logoIconUrl: "../../ms/uiframework/resource/uicommons/images/logo/openmrs-with-title-small.png",
```

Comment:
Do these really belong on the state, or are they just stand-alone variables that could be directly defined on the object or elsewhere?

Reply:
Or, actually, could they just be the default value of the props?

Reply:
Fixed